contracts/coconsensus/Coconsensus.sol

```diff
@@ -46,23 +46,173 @@ contract Coconsensus is MasterCopyNonUpgradable, GenesisCoconsensus, MosaicVersi
     }
 
 
+    /* Constants */
+
+    /**
+     * Sentinel pointer for marking the ending of circular,
+     * linked-list of genesis metachain ids.
+     */
+    bytes32 public constant SENTINEL_METACHAIN_ID = bytes32(0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff);
```

Comment:
Is this the constant we ve used previously for sentinel of bytes32? For address sentinel we ve used 0x00..01

Reply:
yes we are using `0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff` for `bytes32`